spec/support/work_mapping_fixtures.rb

```diff
@@ -121,6 +123,7 @@ def request_dro_with_structural_fixture
                             })
   end
 
+  # rubocop:disable Metrics/AbcSize
```

Comment:
Consider moving to line 127 and dropping the explicit disable on line 152, but this is minor.